src/components/InfoBox/InfoBox.tsx

```diff
@@ -2,12 +2,12 @@ import classnames from 'classnames';
 import React, { ReactNode } from 'react';
 import Icon from '../Icon/Icon';
 
-interface InfoBoxProps extends Omit<React.HTMLProps<HTMLDivElement>, 'className'> {
+interface InfoBoxProps extends Omit<React.HTMLProps<HTMLDivElement>, 'className' | 'title'> {
   children?: ReactNode;
   className?: string;
   color?: string;
   icon?: string;
-  title?: string;
+  title?: string | Element;
```

Comment:
Should this be `ReactNode` or `Element`?

Reply:
Our use case is just an `Element` but curious if `ReactNode` will work for basic HTML elements as well and if that is better for future use cases?

Reply:
[`ReactNode`](https://reactnative.dev/docs/react-node) works for all things that might be `children` of a component.

Reply:
Yeah actually I do need it